EventFilter/L1TRawToDigi/plugins/implementations_stage2/CaloTowerUnpacker.cc

```diff
@@ -18,7 +18,7 @@ namespace stage2 {
      unsigned int bxid = block.amc().getBX();
 
      // handle offset between BC0 marker and actual BC0...
-     if( (tmt-1) != ((bxid-1+3)%9) ) return true;
+     //if( (tmt-1) != ((bxid-1+3)%9) ) return true;
```

Comment:
Yes, the functionality is moved to the proper place, form Calo- and MP-unpacker to  L1TRawToDigi. The unpacker is smarter and makes the TMT node check, and ensures that only the MP that processed a given BX is unpacked.
Will remove obsolete commented lines.